caravel/assets/javascripts/explorev2/components/ExploreViewContainer.jsx

```diff
@@ -3,24 +3,45 @@ import ChartContainer from './ChartContainer';
 import ControlPanelsContainer from './ControlPanelsContainer';
 import QueryAndSaveButtons from './QueryAndSaveButtons';
 
-const ExploreViewContainer = function () {
-  return (
-    <div className="container-fluid">
-      <div className="row">
-        <div className="col-sm-3">
-          <QueryAndSaveButtons
-            canAdd="True"
-            onQuery={() => { console.log('clicked query'); }}
-          />
-          <br /><br />
-          <ControlPanelsContainer />
-        </div>
-        <div className="col-sm-9">
-          <ChartContainer />
+export default class ExploreViewContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      height: this.getHeight(),
+    };
+  }
+
+  getHeight() {
+    const navHeight = 90;
+    return (window.innerHeight - navHeight) + 'px';
```

Comment:
do we use template literals anywhere or is this just a preference you have? 
`${window.innerHeight - navHeight}px`